src/main/java/com/epam/ta/reportportal/filesystem/DataStore.java

```diff
@@ -23,6 +23,8 @@
  */
 public interface DataStore {
 
+	String SECRET_INTEGRATION_SALT = "/keystore/secret-integration-salt";
```

Comment:
@avarabyeu @ihar-kahadouski  is it a good idea to provide this path as an environment variable? is it a good idea to use this salt as a signing key fro jwt token as well?

Reply:
i believe it would be useful to be able to override the path over environment variable. Up to you. 
The salt for JWT and signing key definitely must be different values/vars/paths/etc

Reply:
@avarabyeu got it, so, the JWT signing key is left in database
@ihar-kahadouski let's do salt as an env variable

Reply:
done